src/ontology/cl-edit.owl

```diff
@@ -28105,6 +28102,35 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:24855283"^^xsd:string) r
 AnnotationAssertion(rdfs:label obo:CL_4030017 "epithelial cell of late distal convoluted tubule")
 EquivalentClasses(obo:CL_4030017 ObjectIntersectionOf(obo:CL_0000066 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0005102)))
 
+# Class: obo:CL_4030019 (kidney connecting tubule intercalated cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:9890303"^^xsd:string) obo:IAO_0000115 obo:CL_4030019 "An epithelial cell that is part of the connecting tubule with important roles in the regulation of acid-base homeostasis."^^xsd:string)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> obo:CL_4030019 <https://orcid.org/0000-0002-1773-2692>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4030019 "2022-06-29T11:25:22Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:CL_4030019 "kidney connecting tubule intercalated cell")
+EquivalentClasses(obo:CL_4030019 ObjectIntersectionOf(obo:CL_0005010 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0005097)))
+SubClassOf(obo:CL_4030019 obo:CL_1000768)
+
+# Class: obo:CL_4030020 (kidney connecting tubule alpha-intercalated cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:2254448"^^xsd:string) Annotation(oboInOwl:hasDbXref "PMID:24937428"^^xsd:string) Annotation(oboInOwl:hasDbXref "PMID:9890303"^^xsd:string) obo:IAO_0000115 obo:CL_4030020 "A kidney connecting tubule intercalated cell that secretes protons via an apical H+-ATPase and H+/K+ exchanger and reabsorbs bicarbonate by a Cl-/HCO3- exchanger (AE1) located in the basolateral plasma membrane."^^xsd:string)
```

Comment:
The def here should reflect the changes you made in the equiv class - so 
"A renal alpha-intercalated cell that is part of the renal connecting tubule"